dev/TabView/TabView.vcxitems

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <ItemsProjectGuid>{B9F81FEF-1E8D-4FE1-A46B-7002D4C109D2}</ItemsProjectGuid>
+  </PropertyGroup>
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(MSBuildThisFileDirectory)</AdditionalIncludeDirectories>
+    </ClCompile>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <ProjectCapability Include="SourceItemsFromImports" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="$(MSBuildThisFileDirectory)TabView.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)TabViewItem.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)TabViewAutomationPeer.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="$(MSBuildThisFileDirectory)..\Generated\TabView.properties.cpp" />
+    <ClCompile Include="$(MSBuildThisFileDirectory)..\Generated\TabViewItem.properties.cpp" />
+    <ClCompile Include="$(MSBuildThisFileDirectory)TabView.cpp" />
+    <ClCompile Include="$(MSBuildThisFileDirectory)TabViewItem.cpp" />
+    <ClCompile Include="$(MSBuildThisFileDirectory)TabViewAutomationPeer.cpp" />
+  </ItemGroup>
+  <ItemGroup Condition="$(BuildLeanMuxForTheStoreApp) != 'true'">
+    <Page Include="$(MSBuildThisFileDirectory)TabView.xaml">
```

Comment:
shouldn't need this since the overall vcxitems is not included for lean build, right? (I'm trying to restrict usage of that condition to the MUX vcxproj only going forward)

Reply:
Removed.